app/views/steps/application/check_your_answers/edit.html.erb

```diff
@@ -53,10 +53,21 @@
       </summary>
       <div class="govuk-details__text">
         <%=t '.details.text_html' %>
-        <%= link_button t('.details.download_draft_button'), steps_completion_summary_path(format: :pdf),
-                        class: 'ga-pageLink govuk-button--secondary', download: '',
-                        data: { module: 'govuk-button', ga_category: 'check your answers', ga_label: 'download draft' } %>
-
+        <p id="draftGeneratingMessage">
+          <%= t('.details.generating_message') %>
+        </p>
+        <%= link_button t('.details.download_draft_button'),
```

Comment:
just a question can we just link to the file (controoler) without having this link with so many attributes?